meta-resin-common/classes/binary-compress.bbclass

```diff
@@ -0,0 +1,28 @@
+# e.g.  COMPRESS_BINARY = "path/bin1 path/bin2 path/bin3"
```

Comment:
You didn't understand. I wanted to say that in the docker recipe you define absolute paths while in this example you do relatives paths.